test.sh

```diff
@@ -20,15 +20,13 @@ if [[ $OS == "fedora" ]]; then
   PKG_EXTRA="dnf-plugins-core git-core python$PYTHON_VERSION-koji"
   BUILDDEP="dnf builddep"
   PYTHON="python$PYTHON_VERSION"
-  PYTEST="py.test-$PYTHON_VERSION"
 else
   PIP_PKG="python-pip"
   PIP="pip"
   PKG="yum"
   PKG_EXTRA="yum-utils git-core koji python-dockerfile-parse"
   BUILDDEP="yum-builddep"
   PYTHON="python"
-  PYTEST="py.test"
```

Comment:
Why these two deletions?

Reply:
There is no need to track which Py version are we using here, as there will always be just one pytest installed and it will be `/usr/bin/pytest` binary

Reply:
gotcha